dask_cuda/benchmarks/utils.py

```diff
@@ -366,6 +366,11 @@ def parse_benchmark_args(
         type=int,
         help="Number of warmup runs",
     )
+    parser.add_argument(
+        "--gather-shuffle-stats",
+        action="store_true",
+        help="Gather shuffle statistics",
+    )
```

Comment:
Arguments here are have to be supported by all benchmarks, including CuPy ones. This should be moved into `local_cudf_shuffle.py`. Additionally, please make it clear in the help that this is exclusive for `RapidsMPF` and that there's a check to raise a `ValueError` if this is specified when `backend != "rapidsmpf"`.